### PR TITLE
Bug 1306518, Added note on Kubernetes and Docker versions in OSE 3.2

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1306518

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @vikram-redhat PTAL. Thanks!

**Comment by adellape** [comment]
@ahardin-rh I think noting the kube version in an Arch topic would make more sense in:

https://docs.openshift.com/enterprise/latest/architecture/infrastructure_components/kubernetes_infrastructure.html

or 

https://docs.openshift.com/enterprise/latest/architecture/index.html

Probably the Docker version as well, I guess.

To the comment in the BZ about the docs not currently noting the version of kube or docker in use, when there are upgrades to underlying components (like kube or etcd or the registry version), they have usually been noted in the release notes. For example in 3.0:

https://docs.openshift.com/enterprise/3.0/whats_new/ose_3_0_release_notes.html#ose-3-0-1-0
https://docs.openshift.com/enterprise/latest/release_notes/ose_3_1_release_notes.html#ose-3-1-1-enhancements

For the required version of Docker itself on the hosts, that's noted in the Preqreqs:

https://docs.openshift.com/enterprise/latest/install_config/install/prerequisites.html

But, yea, no reason that stuff couldn't also be noted somewhere in the Arch topics.

**Comment by ahardin-rh** [comment]
@adellape Moved the note to https://docs.openshift.com/enterprise/latest/architecture/infrastructure_components/kubernetes_infrastructure.html

**Comment by ahardin-rh** [comment]
@vikram-redhat PTAL. Thanks!

**Comment by ahardin-rh** [comment]
Extra space removed.

**Comment by ahardin-rh** [comment]
@adellape @vikram-redhat , okay to merge? Thanks!

File: architecture/infrastructure_components/kubernetes_infrastructure.adoc [hunk 1]
```diff
@@ -20,6 +20,11 @@ A Kubernetes cluster consists of one or more masters and a set of nodes. You can
 optionally configure your masters for link:#high-availability-masters[high
 availability] (HA) to ensure that the cluster has no single point of failure.
 
+[NOTE]
+====
+OpenShift Enterprise 3.2 uses Kubernetes 1.2 and Docker 1.9.
```
Comment (adellape) [inline architecture/infrastructure_components/kubernetes_infrastructure.adoc]: Need some ifdefs here.

**Comment by ahardin-rh** [comment]
Added ifdefs

**Comment by ahardin-rh** [comment]
Confirmed usage of {product-title} {product-version} in last night's team meeting.

**Comment by adellape** [comment]
:+1:

**Comment by vikram-redhat** [comment]
@ahardin-rh @adellape - Eric Rich created this support matrix: https://access.redhat.com/articles/2176281. Should we link to it as well?

**Comment by adellape** [comment]
Once that's fixed up and published, could be appropriate to link to from somewhere like:

https://docs.openshift.com/enterprise/latest/install_config/install/prerequisites.html

and/or:

https://docs.openshift.com/enterprise/latest/architecture/index.html

At the very least, we should be adding a link to it from the product page.

**Comment by ahardin-rh** [comment]
**revhistory note**

    (for link:../architecture/infrastructure_components/kubernetes_infrastructure.html[Kubernetes Infrastructure])

Added a note that OSE 3.2 uses Kubernetes 1.2 and Docker 1.9.

**Pull request merged**